data/presets/highway/cycleway/traffic_island_shared.json

```diff
@@ -0,0 +1,36 @@
+{
+    "icon": "temaki-pedestrian_and_cyclist",
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "cycleway": "traffic_island"
+    },
+    "addTags": {
+        "highway": "cycleway",
+        "cycleway": "traffic_island"
+    },
+    "reference": {
+        "key": "cycleway",
+        "value": "traffic_island"
+    },
+    "terms": [
+        "crossing island"
+    ],
+    "aliases": [
+        "Crosswalk Island",
+        "Traffic Island",
+        "Pedestrian Island"
+    ],
+    "name": "Cycle & Foot Refuge Island",
+    "locationSet": {
+        "exclude": [
+            "fr",
+            "lt",
+            "pl",
+            "de",
+            "il",
+            "ps"
+        ]
+    }
```

Comment:
I think this should be moved to the top to sync it with data/presets/highway/cycleway/bicycle_foot.json.